optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/solution/cloner/FieldAccessingSolutionCloner.java

```diff
@@ -419,33 +423,32 @@ protected Object cloneArray(Class<?> expectedType, Object originalArray) {
 
         /**
          * Fails fast if {@link #isFieldAnEntityPropertyOnSolution} assumptions were wrong.
+         *
          * @param originalSolution never null
          * @param cloneSolution never null
          */
         protected void validateCloneSolution(Solution_ originalSolution, Solution_ cloneSolution) {
-            for (MemberAccessor memberAccessor
-                    : solutionDescriptor.getEntityMemberAccessorMap().values()) {
+            for (MemberAccessor memberAccessor : solutionDescriptor.getEntityMemberAccessorMap().values()) {
                 Object originalProperty = memberAccessor.executeGetter(originalSolution);
                 if (originalProperty != null) {
                     Object cloneProperty = memberAccessor.executeGetter(cloneSolution);
                     if (originalProperty == cloneProperty) {
                         throw new IllegalStateException(
                                 "The solutionProperty (" + memberAccessor.getName() + ") was not cloned as expected."
-                                + " The " + FieldAccessingSolutionCloner.class.getSimpleName() + " failed to recognize"
-                                + " that property's field, probably because its field name is different.");
+                                        + " The " + FieldAccessingSolutionCloner.class.getSimpleName() + " failed to recognize"
+                                        + " that property's field, probably because its field name is different.");
                     }
                 }
             }
-            for (MemberAccessor memberAccessor
-                    : solutionDescriptor.getEntityCollectionMemberAccessorMap().values()) {
+            for (MemberAccessor memberAccessor : solutionDescriptor.getEntityCollectionMemberAccessorMap().values()) {
```

Comment:
@rsynek Another example. This doesn't look harmful but it's a different type of place where breaking a line is not allowed.